packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6960,6 +6960,25 @@ class Semantics extends SingleChildRenderObjectWidget {
 /// would be presented as a separate feature than the checkbox, and
 /// the user would not be able to be sure that they were related.
 ///
+/// {@tool snippet}
+/// This snippet shows how to use [MergeSemantics] to merge the semantics of
+/// a [Checkbox] and [Text] widget.
+///
+/// ```dart
+/// MergeSemantics(
+///   child: Row(
+///     children: [
+///       Checkbox(
+///         value: true,
+///         onChanged: (value) {},
```

Comment:
type the value and use arrow method for one liner

(bool value) => null,

Reply:
Ugh, sorry for the nits. Should have done a double check. I have fixed them. Thanks for quick review :)